src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js

```diff
@@ -1008,28 +1008,6 @@ window.bp = window.bp || {};
 							response.data.menu_order = $( file.previewElement ).closest( '.dropzone' ).find( file.previewElement ).index() - 1;
 							self.media.push( response.data );
 							self.model.set( 'media', self.media );
-
-							var image = $( file.previewElement ).find( '.dz-image img' )[0];
-							var isLoaded = image.complete && image.naturalHeight !== 0;
```

Comment:
@bb-yudhisthir Revert this function. This is required for error message display in activity post form.
just do below changes and verify it
var isLoaded = image.complete;